### PR TITLE
Added tabs to navigate between site pages

### PR DESCRIPTION
**Pull request by NicolasDontigny**
image

### CODE REVIEW

File: canopeum_frontend/src/App.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@ $secondary: #F18200;
 $border-radius: 0.5rem;
 $link-decoration: none;
 $green: #06C270;
+$gray: #A2B1A5;
```
Comment (Samuel-Therrien-Beslogic) [inline canopeum_frontend/src/App.scss:8]:
```suggestion
/* TODO: https://github.com/BesLogic/releaf-canopeum/issues/249 */
$green: #06C270;
$gray: #A2B1A5;
```

Comment (Samuel-Therrien-Beslogic) [inline canopeum_frontend/src/App.scss:8]: Let's mark this as TODO with the related issue

```suggestion
/* TODO: https://github.com/BesLogic/releaf-canopeum/issues/249 */
$green: #06C270;
$gray: #A2B1A5;
```

Also reminder to use S2 color + transparency as mentioned on Teams https://teams.microsoft.com/l/message/19:3v6LLSxADDE_UGRewubhTGhsctwyNAjpx6Ar0_AvRwo1@thread.tacv2/1727378057662?tenantId=cd34c2c6-a4f8-4af1-bdd5-497e97626cae&groupId=167b0d82-f8ec-429e-944f-45e9c370f2e3&parentMessageId=1727377426478&teamName=Beslogic%20-%20ARBRES%20Philanthropy&channelName=General&createdTime=1727378057662

**Review by Samuel-Therrien-Beslogic — CHANGES_REQUESTED** [review]

File: canopeum_frontend/src/App.scss (hunk 1 above)
Comment (NicolasDontigny) [reply canopeum_frontend/src/App.scss]: Done!

File: canopeum_frontend/src/App.scss [hunk 2]
```diff
@@ -5,7 +5,9 @@ $secondary: #F18200;
 $border-radius: 0.5rem;
 $link-decoration: none;
 $green: #06C270;
+/* TODO: https://github.com/BesLogic/releaf-canopeum/issues/249 */
```
Comment (Samuel-Therrien-Beslogic) [inline canopeum_frontend/src/App.scss:8]:
```suggestion
/* TODO: We should reference palette, not direct colors
https://github.com/BesLogic/releaf-canopeum/issues/249 */
$green: #06C270;
```

**Review by Samuel-Therrien-Beslogic — APPROVED** [review]

**Pull request merged**